static/src/javascripts/es6/projects/common/modules/crosswords/main.js

```diff
@@ -368,6 +368,23 @@ var Crossword = React.createClass({
         let focussed = this.clueInFocus(),
             isHighlighted = (x, y) => focussed ? helpers.entryHasCell(focussed, x, y) : false;
 
+        // Deep equal version of _.intersection
+        let findIntersectingCells = (array1, array2) =>
```

Comment:
I think we should probably prefer `const` as the default way of declaring things. That way when you see `let` you know that it's specifically something that's being mutated at some point.